src/main/java/com/initializer/Employeedetails/ControlApi/Get.java

```diff
@@ -0,0 +1,94 @@
+package com.initializer.Employeedetails.ControlApi;
+
+import com.initializer.Employeedetails.MessageUtil.MessageUtil;
+import com.initializer.Employeedetails.Repositories.EmployeeRepo;
+import com.initializer.Employeedetails.Repositories.RelationRepo;
+import com.initializer.Employeedetails.Tables.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/******************Service class for retrieving records through ID******************/
+
+@Service
+public class Get
+{   @Autowired
+    EmployeeRepo repository;
+    @Autowired
+    RelationRepo repository1;
+    @Autowired
+    MessageUtil messageUtil;
+
+    public ResponseEntity get(@PathVariable("id") int id)
+    {
+        if(id<=0)
+        {
+            return new ResponseEntity(messageUtil.getMessage("invalid_id"), HttpStatus.BAD_REQUEST);
+        }
+
+        Map<String,Object> mp=new LinkedHashMap<>();
```

Comment:
better not to use map as request/response objects. We lose type safety provided by java